src/engine/UI/menu/Menu.hpp

```diff
@@ -19,6 +21,7 @@ class Menu : public IDisplayLayer {
 
 private:
     std::string m_menuTitle;
```

Comment:
I would argue that we can remove this and replace with a getter that returns the text value of the title option instead.